### DISCUSSION TITLE
duplicate data in ABCD 5.0 Quality Control - Recommended Image Inclusion data (mri_y_qc_incl)

### DISCUSSION THREAD

**Discussion by phoebeycchen** [now-i-know-my-abcd/docs#60] [Where in the World is…? (Help)] [answered]
Dear ABCD team:

I was using the ABCD 5.0 Quality Control - Recommended Image Inclusion data (mri_y_qc_incl) and found that there are duplicates for subject NDAR_INV2F729N9A. Moreover, there are different values for the nback fMRI task (imagincl_nback_include). Please check and let me know which one is correct. Thank you very much!

I also noticed that the 4-yr follow-up recommended inclusion data has not been included in the file. It would be great if you could let me know when to expect this dataset to be released. And what would you suggest I do to deal with the imaging data from the 4-yr follow-up without the information about recommended image inclusion?

I look forward to your reply. Thank you very much!

Kind regards,
Phoebe Chen

**Comment by divyanganar** [comment, marked as answer]
Try using release 5.1 - it includes the 4-year data as well.
https://wiki.abcdstudy.org/release-notes/start-page.html

**Reply by phoebeycchen** [reply]
Thank you so much for the information.
